libraries/AP_Mount/AP_Mount_Servo.cpp

```diff
@@ -169,11 +169,11 @@ void AP_Mount_Servo::stabilize()
 // closest_limit - returns closest angle to 'angle' taking into account limits.  all angles are in degrees * 10
 int16_t AP_Mount_Servo::closest_limit(int16_t angle, int16_t angle_min, int16_t angle_max)
 {
-    // Make sure the angle lies in the interval [-180 .. 180[ degrees
```

Comment:
The [ means that 180 is not inclusive. So please revert this changes, the code was already OK

Reply:
If that's the case, id rather see a more universal sign such as < vs <= to be absolutely clear.

Reply:
I personally learned that [] was inclusive whereas () was not inclusive. So [-180,180) meant the range of -180 inclusive to 180 not inclusive. It sounds like everybody is familiar with different notations.

Reply:
yes, sadly it looks like there are different math notations around the world :(

Reply:
Insane is the word that occurs to me. It's the first time I hear about this other notation. According to https://en.wikipedia.org/wiki/ISO_31-11, *(a, b)* means both an open interval and an ordered pair; smart...

If these comments aren't universal we should change them, but the current change is wrong. Actually these `while`s should go away and `wrap_180_cd` used instead.